web/src/inbox_ui.js

```diff
@@ -39,6 +39,7 @@ const COLUMNS = {
     COLLAPSE_BUTTON: 0,
```

Comment:
<img width="599" alt="Screenshot 2023-09-19 at 10 54 08 PM" src="https://github.com/zulip/zulip/assets/25124304/ef51d660-3dea-47f5-82d4-ccd4076872b2">

Reply:
For some reason, the "mark as unresolved" option makes the item disappear from the inbox view; I'm not sure what the cause is, but I can confirm it happens regardless of what UI is used to unresolve the topic, so I think it's a bug in the inbox view live-update, not in the menu code here.

Reply:
Fixed in #26822